schema.sql

```diff
@@ -17,3 +17,41 @@ CREATE TABLE animals (
 
 ALTER TABLE animals
 ADD COLUMN species varchar(255);
+
+/* Day 3 */
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR,
+    age INTEGER
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR
+);
```

Comment:
- [ ] For the syntax for creating Table "Species" you don’t specify a maximum length when declaring a `VARCHAR` column, the behavior depends on the database system:

-  In SQL Server, if you don’t specify a size for VARCHAR, it defaults to a length of 1.
- In PostgreSQL, if you don’t specify a size for VARCHAR, it can store text up to the system’s maximum allowed length. So, in your second syntax, name VARCHAR in PostgreSQL would allow storing a string of any length up to the system’s limit. However, it’s generally recommended to specify a maximum length as a form of constraint on your data. This can help prevent errors and maintain data integrity.

Reply:
I will fixed it. Thank you